src/content/en/2020/accessibility.md

```diff
@@ -122,7 +122,7 @@ The top 5 file extensions explicitly included in the `alt` text value (for sites
     {{ figure_link(
       caption="Percent of pages with non-empty alt attributes.",
       sheets_gid="TODO",
-      sql_file="TODO.sql"
```

Comment:
@obto could you suggest the correct sheets/sql metadata here? Let's try to resolve these TODOs now if we can.

Reply:
Fixed this.